src/main/java/org/example/PhoneBook.java

```diff
@@ -0,0 +1,174 @@
+package org.example;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class PhoneBook {
```

Comment:
methods in this class are little bit hard to read. There is a general rule in clean coding, to add blank lines between different logical units inside a method.